packages/store/src/internal/dispatcher.ts

```diff
@@ -96,14 +106,13 @@ export class InternalDispatcher {
     return actionResult$
       .pipe(
         exhaustMap((ctx: ActionContext) => {
-          switch (ctx.status) {
-            case ActionStatus.Successful:
-              return of(this._stateStream.getValue());
-            case ActionStatus.Errored:
-              return throwError(ctx.error);
-            default:
-              return empty();
+          if (ctx.status === ActionStatus.Successful) {
```

Comment:
Why has this been changed from the switch to the if? Please change it back.

Reply:
@markwhitfeld Isn't that the same behavior, except `if-else` doesn't lead to nesting like `switch-case`?

Reply:
https://github.com/ngxs/store/issues/739
https://github.com/ngxs/store/issues/487

in that we can now control the management of the zone

This feature will allow you to disable extra detect changes in angular.

Reply:
revert by https://github.com/ngxs/store/pull/763